src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1949,6 +1949,7 @@
 
   <AbstractNode Name="BoundPattern" Base="BoundNode">
     <Field Name="InputType" Type="TypeSymbol" Null="disallow"/>
+    <Field Name="ConvertedType" Type="TypeSymbol" Null="disallow"/>
```

Comment:
Isn't this the same as *output type*? If so we could probably unify.

Reply:
We will unify when the two branches merge.  The spec doesn't use either term.